### DISCUSSION TITLE
start a project in C++ , windows

### DISCUSSION THREAD

**Discussion by DavidBrillianetor** [carla-simulator/carla#6663] [Q&A]
Hello,
I was checking the c++ build guide and saw that there are steps only for linux, 
i would like to start a project in windows with c++ code.
how can i connect to carla server with a client, for start ?
I was checking the code in the link below but had a hard time understanding what are the imports in the code refer to, after trying to run the script i got an error message: 
**cannot open source file "rpc/msgpack.hpp"**

It seems that im missing some files.. but i have download carla simulator currectly..
([https://github.com/carla-simulator/carla/blob/master/Examples/CppClient/main.cpp]
thanks.

**Comment by DavidBrillianetor** [comment]
Found an answer for the question..

**Reply by rohitrnath** [reply]
how did you resolve this?
